src/Packetery/Core/Interfaces/IValidatorTranslations.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class IValidatorTranslations
+ *
+ * @package Packetery
+ */
+
+declare( strict_types=1 );
+
+namespace Packetery\Core\Interfaces;
+
+/**
+ * Class IValidatorTranslations
+ *
+ * @package Packetery
+ */
+interface IValidatorTranslations {
```

Comment:
Prosím nepojmenovávat interface s prefixem "I". Vím, že to tak používá nette, tak to asi k tomu svádí, ale už jen že je to I zkratka a zkratky všeobecně moc používat nechceme a také je to proti PSR. Interface by měl mít vždy suffix tj. ValidatorTranslationsInterface viz. https://www.php-fig.org/bylaws/psr-naming-conventions/

Reply:
Přemýšlím ještě, jestli víc zevšeobecnit ten název toho interfacu, aby se dal případně použít i pro jiné překlady nežli jen u validatoru, ale to je zase takové programování kdyby co by, protože to jinde momentálně nepoužíváme.

Reply:
Proti názvu ValidatorTranslationsInterface místo IValidatorInterface nic nemám.